app/(app)/onboarding/onboarding.tsx

```diff
@@ -0,0 +1,40 @@
+// App.js
+import React from 'react';
+
+function App() {
+  return (
+    <div className="container mx-auto px-4">
+      <h1 className="text-6xl text-black">Welcome!</h1>
+      <p className="text-2xl text-purple-600">Let's sign up</p>
+      <div className="text-boxes-container space-y-5">
+        <div className="w-80">
+          <select name="experience" id="experience" className="input input-primary input-bordered w-full">
```

Comment:
https://daisyui.com/components/select/

Take a look at the select documentation for daisyui. If you click on the "JSX" table, you can see how they make their components:
![image](https://github.com/ai-interviews/web/assets/29685125/4941d532-203a-48aa-8969-58b501e1a407)